package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "dependencies": {
```

Comment:
Do we need whole `plantingjs` stack here? 
Also it's good idea to separate dev dependencies from dependencies (http://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies) - we can create separate issue for that.